src/components/fields/TagArrayField.js

```diff
@@ -128,15 +135,17 @@ export class TagInputComponent extends React.Component {
 			onKeyDown: this.onKeyDown
 		};
 
+		const {showDeleteButton = true, onTagClick} = getUiOptions(uiSchema);
+
 		return (
 			<div className={`rw-multiselect rw-widget${this.state.focused ? " rw-state-focus" : ""}${readonly || disabled ? " rw-state-disabled" : ""}`}
 				onClick={this.onClick}>
 				<div className="rw-widget-input rw-widget-picked rw-widget-container">
 					<ul className="rw-multiselect-taglist">
 						{tags.map((item, idx) => 
-							<li key={idx} className="rw-multiselect-tag">
+							<li key={idx} className="rw-multiselect-tag" onClick={() => onTagClick?.(idx)}>
```

Comment:
Same thing as the event handlers of the above comment: don't use arrow functions as event handlers, bind it to the class instance instead.